DependencyInjection/Configuration.php

```diff
@@ -245,36 +245,35 @@ public function getConfigTreeBuilder()
                     ->end()
                 ->end()
                 ->arrayNode('admin_services')
-                    ->prototype('array')
```

Comment:
Uuuuuh why did you remove this?

Reply:
```
$rootNode = $treeBuilder->root('sonata_admin', 'array');
```
this already defining the global node type,  and mentioned **->prototype('array')** breaks underlying nodes, configuration in yml doesn't work and throws that error as in #3190 .

Reply:
I think you are removing the ability to set admin services on an admin per admin basis when doing that, aren't you? Why not create a `default_admin_service` node instead, and not touch that? This looks like a BC-break.